superset-frontend/src/dashboard/components/FiltersBadge/DetailsPanel.tsx

```diff
@@ -100,6 +101,20 @@ const DetailsPanelPopover = ({
 
   const content = (
     <Panel>
+      <Global
+      styles={css`
+        .ant-popover-inner {
+          background-color: ${theme.colors.grayscale.dark2}cc;
+        }
+        .ant-popover > .ant-popover-content > .ant-popover-arrow{
+          border-top-color: ${theme.colors.grayscale.dark2}cc;
+          border-left-color: ${theme.colors.grayscale.dark2}cc;
```

Comment:
This would need to be more sophisticated to deal with popovers that open upwards.

Reply:
Done and done.